hack/common.sh

```diff
@@ -1001,6 +1001,23 @@ function os::build::find-binary() {
 }
 readonly -f os::build::find-binary
 
+# os::build::get-go-bin-location-from-gopath locates a go install binary in GOPATH directories 
+# and returns the first path where the binary was found. 
```

Comment:
Since this function both takes a parameter and reads some global environment state, please explain that in this comment. An example of how to do that is [here](https://github.com/openshift/origin/blob/master/hack/lib/util/misc.sh#L8-L11).